### PR TITLE
712 update secretless yml test cases

### PR DESCRIPTION
**Pull request by jonahx**
#### What ticket does this PR close?
Connected to [relevant GitHub issues, eg #76]
#### What is the status of the manual tests?
Have you run the following manual tests to verify existing functionality continues to function as expected?
- [ ] Manually tested [K8s CRDs](https://github.com/cyberark/secretless-broker/tree/master/test/manual/k8s_crds)
- [ ] Manually tested [Keychain provider](https://github.com/cyberark/secretless-broker/tree/master/test/manual/keychain_provider)
- [ ] Manually run the [K8s demo](https://github.com/cyberark/secretless-broker/tree/master/demos/k8s-demo)
- [ ] Manually run [Kubernetes-Conjur demo](https://github.com/conjurdemos/kubernetes-conjur-demo) with a local Secretless Broker image build of your branch
- [ ] Manually run the [full demo](https://github.com/cyberark/secretless-broker/tree/master/demos/full-demo) (optional)

If this feature does not have any/sufficent automated tests, have you created/updated a folder in `test/manual` that includes:
- [ ] An updated README with instructions on how to manually test this feature
- [ ] Utility `start` and `stop` scripts to spin up and tear down the test environments
- [ ] A `test` script to run some basic manual tests (optional; if does not exist, the README should have detailed instructions)

### CODE REVIEW

File: cmd/shared/main.go [hunk 1]
```diff
@@ -13,7 +13,7 @@ import (
 	"github.com/cyberark/secretless-broker/internal/app/secretless"
 	"github.com/cyberark/secretless-broker/internal/app/secretless/handlers/mysql/protocol"
 	"github.com/cyberark/secretless-broker/internal/pkg/plugin"
-	"github.com/cyberark/secretless-broker/pkg/secretless/config"
+	"github.com/cyberark/secretless-broker/pkg/secretless/config/v1"
```
Comment (izgeri) [inline cmd/shared/main.go]: can we call this `config_v1` to be consistent with naming elsewhere?

File: internal/app/secretless/configurationmanagers/configfile/manager.go [hunk 2]
```diff
@@ -2,6 +2,8 @@ package configfile
 
 import (
 	"fmt"
+	"github.com/cyberark/secretless-broker/pkg/secretless/config"
+	"github.com/cyberark/secretless-broker/pkg/secretless/config/v1"
```
Comment (izgeri) [inline internal/app/secretless/configurationmanagers/configfile/manager.go]: `config_v1`

File: demos/full-demo/conjur/secretless_ansible.yml [hunk 3]
```diff
@@ -1,11 +1,12 @@
-version: 2
```
Comment (izgeri) [inline demos/full-demo/conjur/secretless_ansible.yml]: I wouldn't worry about this change, I remove this folder `demos/full-demo` in #736

File: demos/full-demo/conjur/secretless_myapp.yml [hunk 4]
```diff
@@ -1,12 +1,18 @@
-version: 2
```
Comment (izgeri) [inline demos/full-demo/conjur/secretless_myapp.yml]: this also gets removed in #736

File: demos/quick-start/docker/etc/secretless.yml [hunk 5]
```diff
@@ -1,40 +1,53 @@
-version: 2
```
Comment (izgeri) [inline demos/quick-start/docker/etc/secretless.yml]: this should be using version 2, but I update it in #736

File: test/plugin/secretless.dev.yml [hunk 6]
```diff
@@ -1,22 +1,34 @@
-version: 2
```
Comment (izgeri) [inline test/plugin/secretless.dev.yml]: this should be using version 2

**Comment by jonahx** [comment]
The changes to all the yml files have already been merged into master.  There are no other relevant changes on the branch.

**Pull request closed without merging**